saleor/graphql/product/mutations/products.py

```diff
@@ -1372,6 +1373,25 @@ def success_response(cls, instance):
         update_product_minimal_variant_price_task.delay(instance.product_id)
         return super().success_response(instance)
 
+    @classmethod
+    def perform_mutation(cls, _root, info, **data):
+        node_id = data.get("id")
+        instance = cls.get_node_or_error(info, node_id, only_type=ProductVariant)
+
+        # get draft order lines for variant
+        line_pks = list(
+            order_models.OrderLine.objects.filter(
+                variant=instance, order__status=OrderStatus.DRAFT
+            ).values_list("pk", flat=True)
```

Comment:
Minor thing but we could use the queryset that we have for drafts - `Order.objects.drafts()`.

Reply:
But we need here `OrderLine` instances not `Order`.

Reply:
Yep, sorry :)